assignment-1/src/main.py

```diff
@@ -1,93 +0,0 @@
-import random
-import inspect
-import cmath
-from task1 import decorator_1
-from task2 import decorator_2
-from task3 import decorator_3
-from task4 import decorator_4, decorator_4_function
```

Comment:
Suggestion: sort your imports.  Add a blank line between external modules and your own.